README.md

```diff
@@ -307,13 +307,14 @@ When using drivers, you will generally want to enable two compilation options (s
 License
 -------
 
-Unless specifically indicated otherwise in a file, Mbed TLS files are provided under the [Apache-2.0](https://spdx.org/licenses/Apache-2.0.html) license. See the [LICENSE](LICENSE) file for the full text of this license. Contributors must accept that their contributions are made under both the Apache-2.0 AND [GPL-2.0-or-later](https://spdx.org/licenses/GPL-2.0-or-later.html) licenses. This enables LTS (Long Term Support) branches of the software to be provided under either the Apache-2.0 OR GPL-2.0-or-later licenses.
+Unless specifically indicated otherwise in a file, Mbed TLS files are provided under the [Apache-2.0](https://spdx.org/licenses/Apache-2.0.html) license. See the [LICENSE](LICENSE) file for the full text of this license, and [the 'License and Copyright' section in the contributing guidelines](CONTRIBUTING.md#License-and-Copyright) for more information.
 
 ### Third-party code included in Mbed TLS
+
 This project contains code from other projects. This code is located within the `3rdparty/` directory. The original license text is included within project subdirectories, and in source files. The projects are listed below:
 
 * `3rdparty/everest/`: Files stem from [Project Everest](https://project-everest.github.io/) and are distributed under the Apache 2.0 license.
```

Comment:
I think we should drop the reference to licences in the third party section. It's confusing and not relevant to say the upstream project is distributed under Apache-2 (we have taken the third party contributions under Apache-2-AND-GPL-2.0-or-later). It's redundant to say that we distribute under Apache-2 (they're distributed under the same terms as the rest of Mbed TLS, which are specified above).

Reply:
Well we explicitly say that the licenses for third party code may be different, so I think it is useful rather than redundant.

And they aren't our code, they are third-party code, kept separately, so I think mentioning it is relevant (and it's less relevant to contributors, so the readme is the right place)